### PR TITLE
Adding all-contributors bot, streamlining Contributing guidelines, also License should display prominently now.

### PR DESCRIPTION
**Pull request by Genesis3800**
Will fix ASAP if README breaks with this PR.

No way to make 100% sure it doesn't beforehand on fork.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "projectName": "availproject.github.io",
+  "projectOwner": "availproject"
+}
```
Comment (salmad3) [inline .all-contributorsrc:4]:
```suggestion
}

```

**Review by salmad3 — APPROVED** [review]

**Review by salmad3 — CHANGES_REQUESTED** [review]
Let's leave the License as "LICENSE" because we only have one License.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+<div align="Center">
+<h1>Contribution Guidelines for Avail's docs</h1>
+</div>
+
+- [How to Contribute to the Avail Documentation](#how-to-contribute-to-the-avail-documentation)
```
Comment (salmad3) [inline CONTRIBUTING.md:5]: We should re-format the heading hierarchy now that these sections are part of a new doc and not part of the heading hierarchy of the README.

**Review by salmad3 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Genesis3800) [reply CONTRIBUTING.md:5]: Done

**Comment by Genesis3800** [comment]
Done @salmad3 ,

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+  - [How to Contribute Changes via the Avail Documentation Website](#how-to-contribute-changes-via-the-avail-documentation-website)
+  - [How to Locally Run the Avail Documentation](#how-to-locally-run-the-avail-documentation)
+    - [Quick Run](#quick-run)
+    - [Setup Steps](#setup-steps)
+
+# How to Contribute to the Avail Documentation
```
Comment (salmad3) [inline CONTRIBUTING.md:11]:
```suggestion
## Ways to Contribute to the Avail Documentation
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+4. **Edit Mode**: Once on GitHub, click the pencil icon located in the upper-right corner to enter edit mode.
+5. **Make Edits**: Modify the Markdown file as needed.
+6. **Initiate Pull Request**: Scroll to the bottom and click on **Create pull request**.
+7. **Title Your PR**: Give your pull request a descriptive title. For example, if you're editing the "Getting Started" page, you could title it _Update /docs/getting-started.md_.
+8. **Describe Changes**: In the pull request description, specify the issue your changes resolve.
+   > See [GitHub Docs on Linking a Pull Request to an Issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for guidance.
+9. **Additional Information**: Provide a concise summary of the changes you've made. Include screenshots or references if applicable.
+10. **Submit**: Click **Propose changes** to finalize your pull request. This will create a new branch in your fork.
+
+### How to Locally Run the Avail Documentation
```
Comment (salmad3) [inline CONTRIBUTING.md:154]: this section is repeated https://github.com/availproject/availproject.github.io/pull/194/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R59

**Review by salmad3 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
+<div align="Center">
+<h1>Contribution Guidelines for Avail's docs</h1>
+</div>
+
+- [How to Contribute to the Avail Documentation](#how-to-contribute-to-the-avail-documentation)
+  - [How to Contribute Changes via the Avail Documentation Website](#how-to-contribute-changes-via-the-avail-documentation-website)
```
Comment (salmad3) [inline CONTRIBUTING.md:6]:
```suggestion
- [How to Contribute Changes via the Avail Documentation Website](#how-to-contribute-changes-via-the-avail-documentation-website)
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+  - [How to Locally Run the Avail Documentation](#how-to-locally-run-the-avail-documentation)
```
Comment (salmad3) [inline CONTRIBUTING.md:7]:
```suggestion
- [How to Locally Run the Avail Documentation](#how-to-locally-run-the-avail-documentation)
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (salmad3) [inline CONTRIBUTING.md:5]:
```suggestion
- [Ways to Contribute to the Avail Documentation](#how-to-contribute-to-the-avail-documentation)
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+    - [Quick Run](#quick-run)
```
Comment (salmad3) [inline CONTRIBUTING.md:8]:
```suggestion
  - [Quick Run](#quick-run)
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+    - [Setup Steps](#setup-steps)
```
Comment (salmad3) [inline CONTRIBUTING.md:9]:
```suggestion
  - [Setup Steps](#setup-steps)
```

File: CONTRIBUTING.md (hunk 4 above)
Comment (Genesis3800) [reply CONTRIBUTING.md:154]: Damn. Missed it. My bad. The whole thing after the LICENSE heading was repeated.

**Review by salmad3 — APPROVED** [review]

**Pull request merged**